pulp_2to3_migration/app/serializers.py

```diff
@@ -156,6 +156,7 @@ class Pulp2RepositoriesSerializer(ModelSerializer):
     )
     pulp2_object_id = serializers.CharField(max_length=255)
     pulp2_repo_id = serializers.CharField()
+    type = serializers.CharField()
```

Comment:
what about pulp2_repo_type to stay consistent with other fields names?

Reply:
+1

Reply:
Should I rename the database field as well?  It is just ```type```

Reply:
+1 to that

Reply:
Would you also like the database field renamed?  It is just ```type```